### PR TITLE
Fixed: Various Translations

### PR DESCRIPTION
**Pull request by bakerboy448**
#### Database Migration
NO

#### Description
A few sentences describing the overall goals of the pull request's commits.

#### Todos
- [ ] Tests
- [x] Translation Keys (./src/NzbDrone.Core/Localization/Core/en.json)
- [ ] [Wiki Updates](https://wiki.servarr.com)

#### Issues Fixed or Closed by this PR

* Fixes #XXXX

### CODE REVIEW

File: src/NzbDrone.Core/Localization/Core/en.json [hunk 1]
```diff
@@ -295,19 +265,19 @@
...
   "ProxyUsernameHelpText": "You only need to enter a username and password if one is required. Leave them blank otherwise.",
   "PtpOldSettingsCheckMessage": "The following PassThePopcorn indexers have deprecated settings and should be updated: {0}",
-  "QualityDefinitions": "Quality Definitions",
-  "QualitySettings": "Quality Settings",
   "Query": "Query",
   "QueryOptions": "Query Options",
+  "QueryResults": "'Query Results'",
   "Queue": "Queue",
+  "Radarr": "Radarr",
+  "Readarr": "Readarr",
```
Comment (Qstick) [inline src/NzbDrone.Core/Localization/Core/en.json:273]: Unused now

File: src/NzbDrone.Core/Localization/Core/en.json [hunk 2]
```diff
@@ -274,13 +247,12 @@
...
   "Password": "Password",
   "Peers": "Peers",
-  "Pending": "Pending",
   "PendingChangesDiscardChanges": "Discard changes and leave",
   "PendingChangesMessage": "You have unsaved changes, are you sure you want to leave this page?",
   "PendingChangesStayReview": "Stay and review changes",
   "Port": "Port",
   "PortNumber": "Port Number",
-  "PreferredSize": "Preferred Size",
+  "Public": "Public",
```
Comment (Qstick) [inline src/NzbDrone.Core/Localization/Core/en.json:255]: Incorrect sort... Sort this back.

Comment (bakerboy448) [reply src/NzbDrone.Core/Localization/Core/en.json]: ahhh swear i sorted before

File: src/NzbDrone.Core/Localization/Core/en.json [hunk 3]
```diff
@@ -441,15 +410,18 @@
   "Updates": "Updates",
   "UpdateScriptPathHelpText": "Path to a custom script that takes an extracted update package and handle the remainder of the update process",
   "Uptime": "Uptime",
+  "Url": "'Url'",
```
Comment (Qstick) [inline src/NzbDrone.Core/Localization/Core/en.json:413]: Why is this in single quotes. I think it was just like that in the original code so it wouldn't interfere with the variable named the same.

File: src/NzbDrone.Core/Localization/Core/en.json [hunk 4]
```diff
@@ -441,15 +410,18 @@
...
   "URLBase": "URL Base",
   "UrlBaseHelpText": "For reverse proxy support, default is empty",
   "Usenet": "Usenet",
   "UseProxy": "Use Proxy",
+  "UserAgentProvidedByTheAppThatCalledTheAPI": "User-Agent provided by the app that called the API",
   "Username": "Username",
   "Version": "Version",
   "View": "View",
   "Warn": "Warn",
+  "Website": "'Website'",
```
Comment (Qstick) [inline src/NzbDrone.Core/Localization/Core/en.json:423]: Same, should this be double quoted?

File: src/NzbDrone.Core/Localization/Core/en.json [hunk 5]
```diff
@@ -399,6 +368,7 @@
   "TestAll": "Test All",
   "TestAllApps": "Test All Apps",
   "TestAllClients": "Test All Clients",
+  "TestAllIndexers": "'Test All Indexers'",
```
Comment (Qstick) [inline src/NzbDrone.Core/Localization/Core/en.json:371]: And this

File: src/NzbDrone.Core/Localization/Core/en.json [hunk 6]
```diff
@@ -300,10 +272,9 @@
   "ProxyUsernameHelpText": "You only need to enter a username and password if one is required. Leave them blank otherwise.",
   "PtpOldSettingsCheckMessage": "The following PassThePopcorn indexers have deprecated settings and should be updated: {0}",
   "Public": "Public",
-  "QualityDefinitions": "Quality Definitions",
-  "QualitySettings": "Quality Settings",
   "Query": "Query",
   "QueryOptions": "Query Options",
+  "QueryResults": "'Query Results'",
```
Comment (Qstick) [inline src/NzbDrone.Core/Localization/Core/en.json:277]: And This

File: src/NzbDrone.Core/Localization/Core/en.json [hunk 7]
```diff
@@ -220,27 +205,16 @@
   "LogLevel": "Log Level",
   "LogLevelTraceHelpTextWarning": "Trace logging should only be enabled temporarily",
   "Logs": "Logs",
-  "MaintenanceRelease": "Maintenance release",
+  "MaintenanceRelease": "Maintenance Release: bug fixes and other improvements. See Github Commit History for more details",
   "Manual": "Manual",
-  "MaximumLimits": "Maximum Limits",
+  "MassEditor": "'Mass Editor'",
```
Comment (Qstick) [inline src/NzbDrone.Core/Localization/Core/en.json:210]: And This

File: src/NzbDrone.Core/Localization/Core/en.json [hunk 8]
```diff
@@ -175,18 +158,21 @@
...
   "Id": "Id",
   "IgnoredAddresses": "Ignored Addresses",
   "IllRestartLater": "I'll restart later",
-  "Importing": "Importing",
   "IncludeHealthWarningsHelpText": "Include Health Warnings",
   "Indexer": "Indexer",
   "IndexerAuth": "Indexer Auth",
   "IndexerFlags": "Indexer Flags",
   "IndexerHealthCheckNoIndexers": "No indexers enabled, Prowlarr will not return search results",
+  "IndexerInfo": "'Indexer info'",
```
Comment (Qstick) [inline src/NzbDrone.Core/Localization/Core/en.json:175]: And This

File: src/NzbDrone.Core/Localization/Core/en.json [hunk 9]
```diff
@@ -61,6 +61,7 @@
   "BypassProxyForLocalAddresses": "Bypass Proxy for Local Addresses",
   "Cancel": "Cancel",
   "CancelPendingTask": "Are you sure you want to cancel this pending task?",
+  "Categories": "'Categories'",
```
Comment (Qstick) [inline src/NzbDrone.Core/Localization/Core/en.json:64]: And This

File: src/NzbDrone.Core/Localization/Core/en.json (hunk 4 above)
Comment (bakerboy448) [reply src/NzbDrone.Core/Localization/Core/en.json:423]: i swear i fixed these already lol

**Review by Qstick — APPROVED** [review]

**Pull request merged**